core/src/main/java/org/stellar/anchor/sep10/Sep10Service.java

```diff
@@ -34,27 +33,35 @@ public class Sep10Service {
 
   public Sep10Service(
       AppConfig appConfig, Sep10Config sep10Config, Horizon horizon, JwtService jwtService) {
+    infoF("Creating Sep10Service");
+    infoConfig("appConfig:", appConfig, AppConfig.class);
     this.appConfig = appConfig;
+
+    infoConfig("sep10Config:", sep10Config, Sep10Config.class);
     this.sep10Config = sep10Config;
+
     this.horizon = horizon;
     this.jwtService = jwtService;
     this.serverAccountId = KeyPair.fromSecretSeed(sep10Config.getSigningSeed()).getAccountId();
   }
 
   public ChallengeResponse createChallenge(ChallengeRequest challengeRequest) throws SepException {
+    info("Creating challenge");
     //
     // Validations
     //
     if (challengeRequest.getHomeDomain() == null) {
+      infoF("home_domain is not specified. Use {}", sep10Config.getHomeDomain());
       challengeRequest.setHomeDomain(sep10Config.getHomeDomain());
     } else if (!sep10Config.getHomeDomain().equalsIgnoreCase(challengeRequest.getHomeDomain())) {
+      infoF("Bad home_domain: {}", challengeRequest.getHomeDomain());
       throw new SepValidationException(
           String.format("home_domain [%s] is not supported.", challengeRequest.getHomeDomain()));
     }
 
     if (sep10Config.isClientAttributionRequired()) {
       if (challengeRequest.getClientDomain() == null) {
-        infoF("ALERT: client domain required and not provided");
+        info("client_domain is required but not provided");
         throw new SepValidationException("client_domain is required");
       }
 
```

Comment:
Please allow me to discuss a concept here: I'd prefer to use `error`, rather than `info` logs when we're inside a `catch`, WDYT?